README.md

```diff
@@ -494,6 +489,11 @@ To add Mantle to your application:
     to add the appropriate Mantle target to the "Target Dependencies" of your
```

Comment:
This step should also be unnecessary, because implicit dependency lookup works within one project too AFAIK.